R/expect_identical_linter.R

```diff
@@ -69,6 +69,10 @@ expect_identical_linter <- function() {
         and expr[NUM_CONST[contains(text(), '.')]]
       ]
       or following-sibling::expr[NUM_CONST[contains(text(), '.')]]
+      or following-sibling::expr[
+        OP-MINUS
+        and expr[NUM_CONST[contains(text(), '.')]]
+      ]
```

Comment:
I see two potential solutions:

```suggestion
      or following-sibling::expr[
        OP-MINUS
        and not(descendant::SYMBOL)
        and expr[NUM_CONST[contains(text(), '.')]]
      ]
```

OR 

```suggestion
      or following-sibling::expr[
        OP-MINUS
        and count(expr) = 1
        and expr[NUM_CONST[contains(text(), '.')]]
      ]
```

The latter would lint cases such as `expect_equal(x, 1.01-0.01)` but maybe that's what we want :thinking:.

Reply:
then `count(expr) = 1` version seems more intuitive.
I think the idea of detecting constant expressions (like `1.004 - 1.003`, `2.0 * 42.0`, `10L + 2^5`) is best done with a more general approach.

Reply:
related: #963